Services/UserService/Controllers/UsersController.cs

```diff
@@ -19,6 +21,16 @@ public class UsersController : ControllerBase
         [HttpPost]
         public async Task<bool> CreateUser([FromServices] ICreateUserCommand command, [FromBody] CreateUserRequest request)
         {
+            CreateUserRequesValidator validator = new CreateUserRequesValidator();

```

Comment:
What? Nononono
We already have data validation in createUserCommand
Why do you have to write all logic here?!